frontend/src/component/FeedItem/FeedItemHeader.jsx

```diff
@@ -148,14 +157,29 @@ export class FeedItemHeader extends React.Component {
               <i className={`fa-fw ${props.contentType.faIcon}`} />
               {props.t('Open as content')}
             </Link>
+
+            {shouldShowChangeTypeButton && (
+              <IconButton
+                customClass='feedItemHeader__actionMenu__item'
+                disabled={props.content.is_archived || props.content.is_deleted}
+                icon='fa-fw fas fa-fw fa-th'
+                text={props.t('Change into content')}
+                textMobile={props.t('Change into content')}
+                label={props.t('Change into content')}
+                key={`change-type-${contentId}`}
+                onClick={props.onClickChangeType}
+                dataCy='popinListItem__change_type'
+              />
```

Comment:
I'd prefer naming "content-type" whether it's change, turn or convert

Reply:
I change popinListItem__change_type to popinListItem__content_type